apis/field_error.go

```diff
@@ -335,3 +335,12 @@ func ErrInvalidKeyName(value, fieldPath string, details ...string) *FieldError {
 		Details: strings.Join(details, ", "),
 	}
 }
+
+// ErrOutOFBoundsValue constructs a FieldError for a field that has received an
+// out of bound value.
+func ErrOutOfBoundsValue(value, lower, upper float64, fieldPath string) *FieldError {
```

Comment:
I think this name is a bit too general for the signature being specifically float64.

Reply:
change it to take string.
ptal